src/deadline/job_attachments/api/manifest.py

```diff
@@ -247,3 +260,191 @@ def process_output(status: FileStatus, path: str, output_diff: ManifestDiff):
             process_output(fast_diff_item[1], fast_diff_item[0], output)
 
     return output
+
+
+def _manifest_upload(
+    manifest_file: str,
+    s3_bucket_name: str,
+    s3_cas_prefix: str,
```

Comment:
WDYT about using `s3_cas_uri` to be consistent with the CLI option?

Reply:
No - because at the API level this is split to bucket name and cas prefix. This small difference is due to customers not always using Queue Job Attachment settings. Programatically, they are 2 things, just like the Queue settings's attributes.